platform/android/export/export.cpp

```diff
@@ -2028,6 +2028,13 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		// Validate the rest of the configuration.
 
 		String dk = p_preset->get("keystore/debug");
+		String dk_user = p_preset->get("keystore/debug_user");
+		String dk_password = p_preset->get("keystore/debug_password");
+
+		if ((dk.is_empty() || dk_user.is_empty() || dk_password.is_empty()) && (!dk.is_empty() || !dk_user.is_empty() || !dk_password.is_empty())) {
+			valid = false;
+			err += TTR("Either Debug keystore, Debug User AND Debug Password presets must be configured OR none of them.") + "\n";
```

Comment:
Use consistent capitalization and terminology:

```suggestion
			err += TTR("Either Debug Keystore, Debug User AND Debug Password settings must be configured OR none of them.") + "\n";
```